components/tutorial/tutorial_tip/tutorial_tip.tsx

```diff
@@ -122,19 +139,19 @@ export default class TutorialTip extends React.PureComponent<Props, State> {
     }
 
     private autoShow(couldAutoShow: boolean) {
-        const {autoTour, currentStep, step, firstChannelName} = this.props;
```

Comment:
why remove that from destructuring?

Reply:
Just removed unnecessary const

Reply:
why is that unnecessary? If you need to use `this.props.X` more than once then it is typically better to destructure `X` from props.